ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/CliCommand.java

```diff
@@ -67,29 +92,107 @@ protected FileUtils getFileUtils() {
         return this.fileUtils;
     }
 
-    public abstract CliCommand init(final Arguments args) throws Exception;
+    /**
+     * Lookup help for '--help' COMMAND from resource file
+     * (Override or add help text to resources/com/spectralogic/dscli/help.properties
+     * @param command
+     * @return
+     */
+    public static String getLongHelp(final String command) {
```

Comment:
Maybe getVerboseHelp() & getConciseHelp() (getToolTipHelp()) or something?  My bandwidth-limited brain interpreted getLongHelp() as get help with a command modifier that is of type long.